config/initializers/secure_headers.rb

```diff
@@ -4,7 +4,7 @@
   config.x_content_type_options = "nosniff"
   config.x_xss_protection = "1; mode=block"
   config.csp = {
-    :enforce     => true,
```

Comment:
Confused; Based on [this](https://github.com/twitter/secureheaders/blob/master/upgrading-to-3-0.md#changes) you seem to be changing this value, as well as the config rename. Is this intended?